eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -47,6 +47,7 @@ jobs:
         - ${{ each pair in step }}:
             ${{ pair.key }}: ${{ pair.value }}
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
```

Comment:
Why not just include the caching in the install-dotnet.yml template?

Reply:
My thinking was that sometimes somebody would want to run a more lightweight task using dotnet that doesn't require 1.7gb cache.

Reply:
That is fair but I guess we should look at the performance or we could add an opt-out template parameter for the install step for folks that want to disable the cache.

Reply:
Moved.